README.md

```diff
@@ -90,4 +111,4 @@ To check locally for broken links, install [`htmltest`](https://github.com/wjdp/
 
 ## Publishing
 
-The docs are automatically published when a PR merges.
+The docs automatically publish when a PR merges to the `main` branch.
```

Comment:
If you want to make it active voice, maybe say something about how a GitHub workflow builds and publishes the docs, since the docs don't quite publish themselves?

Reply:
Great point. I overdid it here, should attribute the action to a GitHub workflow instead:

```suggestion
A GitHub workflow automatically publishes the docs to `https://docs.viam.com/` when new commits appear in the `main` branch.
```